gulpfile.babel.js

```diff
@@ -290,11 +290,11 @@ gulp.task("watch", (cb) => {
 
       // WEBPACK HMR did the job with browser sync
       $.watch(`${paths.site.src}/js/**/*`, () => {
-        runSequence("webpack:lib", "webpack:site");
+        runSequence("webpack:site", "webpack:lib");
       });
 
       $.watch(`${paths.lib.src}/**/*`, () => {
-        runSequence("webpack:lib", "webpack:site");
+        runSequence("webpack:site", "webpack:lib");
```

Comment:
Really? 🤔  
hmmm... this makes sense as we're now importing the es6 module from the _source_ 
```
import Flooper from '../../lib/index.js',
``` 
the previous though was , w8 for new compiled version of lib => then compile the site, not sure what was wrong. Either way, it's working 🎊 :shipit: